qualtran/bloqs/block_encoding/product_test.py

```diff
@@ -150,3 +152,28 @@ def test_product_cirq():
     initial_state = [0, 0, 0]
     final_state = [1, 0, 0]
     assert_circuit_inp_out_cirqsim(circuit, qubits, initial_state, final_state)
+
+
+random.seed(1234)
```

Comment:
Is it safe to randomize globally? iirc, pytest runs multiple tests in the same process, so they'd share the random states. Maybe safer to pass an rng?